### DISCUSSION TITLE
Negative clip

### DISCUSSION THREAD

**Discussion by pstidsen** [anitagraser/movingpandas#354] [Q&A] [answered]
Hi there,

I am wondering if there is a smart way of using clip negatively, getting all the parts of the trajectories which is NOT inside the polygon. I could make a surrounding polygon which e.g. contains whole world but that's a bit odd to do.

Is is possible?

Best Regards,
Peter

**Comment by anitagraser** [comment, marked as answer]
Hi Peter. This is currently not directly possible. So the work-around you mentioned is your best bet.
